test/inspec/author_spec.rb

```diff
@@ -48,13 +52,13 @@
   it { should be_grouped_into 'aem-author' }
 end
 
-describe file(aem_keystore_path) do
-  it { should be_file }
-  it { should exist }
-  its('mode') { should cmp '00640' }
-  it { should be_owned_by 'aem-author' }
-  it { should be_grouped_into 'aem-author' }
-end
+# describe file("#{aem_base}/aem/author/crx-quickstart/ssl/aem.ks") do
+#   it { should be_file }
+#   it { should exist }
+#   its('mode') { should cmp '00640' }
+#   it { should be_owned_by 'aem-author' }
+#   it { should be_grouped_into 'aem-author' }
+# end
```

Comment:
This test is probably still required if jetty is used as ssl method ?

Reply:
Yes, it is required

Reply:
Done